lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -11,12 +11,19 @@ interface UrlMatch {
 }
 
 const urlRegex =
-  /(?:https?:)?(?:\/\/)?(?:www\.)?(?<!api\.)(?:to)?github\.com\/[-_a-z0-9]+\/[-_a-z0-9]+\/(?:discussions|issues|pull)\/[0-9]+(?:#[-_a-z0-9]+)?/i; // TODO #12872 (?<!re) after text not matching
+  /(?:https?:)?(?:\/\/)?(?:www\.)?(?<!api\.)(?:to)?github\.com\/[-_a-z0-9]+\/[a-zA-Z1-9\-_.]+\/(?:discussions|issues|pull)\/[0-9]+(?:#[-_a-z0-9]+)?/i; // TODO #12872 (?<!re) after text not matching
+
+const reduceUrlRegex =
+  /(?:https?:)?(?:\/\/)?(?:www\.)?github\.com\/(?<org>[a-zA-Z1-9\-_.]*)\/(?<repo>[a-zA-Z1-9\-_.]*)\/([a-zA-Z1-9\-_.]*)\/(?<number>[\d]+)/g;
```

Comment:
@viceice 
can't we just use the same URL above and add groups to it, for reducing?  why do we make here another regex

WDYT?

Reply:
i dont see any reason to keep this regex

Reply:
you can try to integrate it. 🤷‍♂️

Reply:
@viceice maroon made this regex because of
`(?<!api\.)`
this is failing validation on `regEx`, so he made a new almost identical regex without it

what is this? `(?<!api\.)` ???

Reply:
removing it

Reply:
it's look ahead regex